src/Microsoft.ML.LightGBM/LightGbmStatic.cs

```diff
@@ -35,6 +35,12 @@ public static partial class RegressionTrainers
         /// the linear model that was trained. Note that this action cannot change the result in any way;
         /// it is only a way for the caller to be informed about what was learnt.</param>
         /// <returns>The Score output column indicating the predicted value.</returns>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[SDCA](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Trainers.cs?range=6-10,135-193 "LightGbm regression example.")]
```

Comment:
>range=6-10,135-193  [](start = 97, length = 19)

check if the range is still relevant. 

Also the first example referenced in SDCA might need to get updated.